sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md

```diff
@@ -0,0 +1,601 @@
+# Kubernetes Admission Control Threat Model
+
+
+[TOC]
```

Comment:
@raesene is this intended to be left empty?

Reply:
good point I think that was just a placeholder from the google docs --> markdown conversion so I removed it.